### DISCUSSION TITLE
Coinbase not in the list of wallets that can be connected

### DISCUSSION THREAD

**Discussion by ShuraRB** [WalletConnect/walletconnect-monorepo#2639] [Developer Support] [answered]
is there gonna be available later?

**Comment by xzilja** [comment, marked as answer]
Coinbase is only available in modals that use `wagmi` package, as they don't use WalletConnect. They are there by default as an extension wallet, but you can also use wagmi's Coinbase connector to use coinbase sdk.

**Reply by ShuraRB** [reply]
In the new version of Wallet Connect, based on my understanding, it seems to indicate that Coinbase will only be available in the web's Web3Modal and not in React Native. Is this correct?
